packages/react-menu/src/components/MenuTrigger/useTriggerElement.ts

```diff
@@ -52,6 +46,16 @@ export const useTriggerElement = (state: UseTriggerElementState): MenuTriggerSta
     child.props?.onClick?.(e);
   });
 
+  const onBlur = useEventCallback((e: React.FocusEvent) => {
+    const isOutsidePopup = !menuPopupRef.current?.contains(e.relatedTarget as Node);
+    const isOutsideTrigger = !triggerRef.current?.contains(e.relatedTarget as Node);
+    if (isOutsidePopup && isOutsideTrigger) {
```

Comment:
A similar code is used in `useMenu()`, do you want to introduce a helper function?

BTW, it uses different naming there `if (isOutsidePopup && isOutside**Target**)`

Reply:
you're right, the naming should be consistent and I'll extract to a helper function